### PR TITLE
feat: update sponsors, backers references

### PR DESCRIPTION
**Pull request by clarkerican**
Removes sponsors, and backers from the Readme

### CODE REVIEW

**Comment by damienwebdev** [comment]
@clarkerican can you split this diff apart? Some of this relates to:

1. README modifications from #28
2. Support for self-managed types #14 
3. Package renaming #19 

And I think we should strive to target one issue at a time.

**Comment by clarkerican** [comment]
@damienwebdev I limited the scope :)

**Review by damienwebdev — APPROVED** [review]
lgtm

File: Readme.md [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
-![Faker.js](https://raw.githubusercontent.com/Marak/faker.js/master/logo.png)
-
-[![Build Status](https://travis-ci.org/Marak/faker.js.svg?branch=master)](https://travis-ci.org/Marak/faker.js) [![Coverage Status](https://coveralls.io/repos/github/Marak/faker.js/badge.svg?branch=master)](https://coveralls.io/github/Marak/faker.js?branch=master)
-
-[![npm version](https://badge.fury.io/js/faker.svg)](http://badge.fury.io/js/faker)
-
-[![OpenCollective](https://opencollective.com/fakerjs/backers/badge.svg)](#backers)
-[![OpenCollective](https://opencollective.com/fakerjs/sponsors/badge.svg)](#sponsors)
-[![Gitter chat](https://badges.gitter.im/marak/faker.js.png)](https://gitter.im/Faker-js/community)
+![Faker.js](https://raw.githubusercontent.com/withshepherd/faker.js/master/logo.png)
```
Comment (Shinigami92) [inline Readme.md:3]: The logo should be served from `faker-js` instead of `withshepherd`

File: Readme.md [hunk 2]
```diff
@@ -413,112 +401,3 @@ npm run browser
...
-
-faker.js - Copyright (c) 2020
-Marak Squires
-www.marak.com
-http://github.com/marak/faker.js/
-
-faker.js was inspired by and has used data definitions from:
-
- * <https://github.com/stympy/faker/> - Copyright (c) 2007-2010 Benjamin Curtis
- * <http://search.cpan.org/~jasonk/Data-Faker-0.07/> - Copyright 2004-2005 by Jason Kohles
```
Comment (Shinigami92) [inline Readme.md:435]: At least we should have this history somewhere.
Maybe when we add a `LICENSE` file we should add these maintainers :thinking:

**Comment by Shinigami92** [comment]
Should we remove `.github/FUNDING.yml` also at least for now?

**Review by Shinigami92 — CHANGES_REQUESTED** [review]
see comments above

File: Readme.md (hunk 2 above)
Comment (damienwebdev) [reply Readme.md:425]: @Shinigami92 this is held within the [LICENSE ](https://github.com/faker-js/faker/blob/main/MIT-LICENSE.txt) file.

Comment (Shinigami92) [reply Readme.md:425]: You are right, thx :+1:

**Review by Shinigami92 — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Review by prisis — APPROVED** [review]

**Review by damienwebdev — APPROVED** [review]
lgtm

**Pull request merged**